library/std/src/sys/pal/unix/thread.rs

```diff
@@ -139,7 +144,7 @@ impl Thread {
                     const TASK_COMM_LEN: usize = 16;
                     let name = truncate_cstr::<{ TASK_COMM_LEN }>(name);
                 } else {
-                    // FreeBSD and DragonFly BSD do not enforce length limits.
+                    // FreeBSD, DragonFly, FreeBSD and NuttX do not enforce length limits.
```

Comment:
Why did you write down FreeBSD twice? :)

Reply:
Oh, sorry, I thought "BSD" referred to "FreeBSD" rather than "Dragonfly BSD". I'll revert it.

Reply:
https://github.com/rust-lang/rust/pull/136052